scripts/lib/python/nas_os_if_utils.py

```diff
@@ -398,9 +426,11 @@ def create_interface_set_obj(if_name, attr_list):
 def physical_ports_for_front_panel_port(fp_obj):
     # replace with object get attr data
     ports = fp_obj.get()['data']['base-if-phy/front-panel-port/port']
-
+    print "ports:", ports
```

Comment:
This function wasn't printing things before; should it be doing so now? Maybe these are debugs left in the PR unintentionally?